bowling_game.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push('Users/andrespacheco/Documents/andres-pacheco/bowling-game-kata-pacheco-andres')
```

Comment:
I won't work in any other computer haha, what about using `require_relative` for importing the classes it depends?

Reply:
I will try that